run_infersharp.sh

```diff
@@ -7,7 +7,7 @@ set -e
 
```

Comment:
do not make changes to the script. it will be a private build for guardian.

Reply:
> do not make changes to the script. it will be a private build for guardian.

Not really sure about what you need me to do to make a private build. Let's sync about details offline.

Reply:
Done